app/models/manageiq/providers/ansible_tower/configuration_manager/configured_system.rb

```diff
@@ -1,8 +1,12 @@
 class ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfiguredSystem < ::ConfiguredSystem
   include ProviderObjectMixin
 
+  belongs_to :configuration_manager,
+             :class_name  => 'ConfigurationManager',
+             :foreign_key => 'configuration_manager_id'
+
   def provider_object(connection = nil)
-    (connection || connection_source.connect).host(manager_ref)
+    (connection || connection_source.connect).api.host(manager_ref)
```

Comment:
I think this should be `(connection || connection_source.connect).api.hosts.find(manager_ref)`, right?